csunplugged/templates/plugging_it_in/results-table.html

```diff
@@ -22,12 +22,30 @@
                 <td>
                     <pre class="mb-0 test-case-output" id="test-case-{{ test_case.id }}-output"></pre>
                 </td>
-                <td id="test-case-{{ test_case.id }}-status">
-                    <span class="text-muted">
+                <td>
+                    <span id="test-case-{{ test_case.id }}-status">
                         Not yet run
                     </span>
+                    <a href="#" id="test-case-{{ test_case.id }}-help-icon" class="badge help-icon" data-toggle="modal" data-target="#modal-{{ test_case.id }}" style="display:none">
```

Comment:
Use bootstraps `d-none` class instead of the inline style :)

Reply:
Good idea, although I just tried it and turns out it overrides the jquery show and hide methods. I've updated the .scss files and .js files accordingly to remove this inline style.